mips/malta.c

```diff
@@ -0,0 +1,64 @@
+#include <mips/malta.h>
+#include <malloc.h>
+
+void clear_bss() {
+  extern unsigned int __bss[];
+  extern unsigned int __ebss[];
+  for (unsigned int *p = __bss, *end = __ebss; p < end; ++p) *p = 0;
+}
+
+#define ISSPACE(a) ((a) == ' ' || ((a) >= '\t' && (a) <= '\r'))
+
+void fix_argv(int *p_argc, char ***p_argv) {
```

Comment:
Can you add a comment that explains what are you trying to fix? Show example input data.